### DISCUSSION TITLE
Exchange Form, field name changed, but field value keeps the same

### DISCUSSION THREAD

**Discussion by nekosgr93** [react-hook-form/react-hook-form#7625] [Q&A] [answered]
Hello, I'm working with some exchange forms(ex: currency or temperature exchange).
The strange thing is that when I changed the scale state, the field name changed but the field value keeps the same.
I don't have any idea why this happened.

Here is a simple temperature exchange form to show the situation.
The log shows that the field name changed, but field value don't.
Thanks for help.
https://codesandbox.io/s/temperature-exchange-uob8q

**Comment by Moshyfawn** [comment, marked as answer]
I'd suggest another approach to make sure the input values stay in sync. See [this CSB](https://codesandbox.io/s/temperature-exchange-forked-n2sww?file=/src/App.tsx).
Basically, collect the measurements values _(fahrenheit & celsius)_ under one value and manage the input changes under one controller.
Take note of how `field.value` are being passed to `<TextInput />`s
